### DISCUSSION TITLE
is it possible to track sources from a main domain to a subdomain using UTM parameters?

### DISCUSSION THREAD

**Discussion by treebetty** [plausible/analytics#1027] [Anything Else]
Hi everyone - is it possible to track sources from a main domain to a subdomain using UTM parameters?

We have our company's website at https://example.com and the app running at https://go.example.com. They're being tracked as two separate sites on Plausible.

We're trying to track sources of conversions using UTM tags. 

If a user clicks on the signup button on the marketing site (www.example.com), we then pass those same UTM tags on to the app site (go.example.com). 

So full example: a user clicks on google ad, then clicks on the signup button on marketing site, and then goes to the app and signs up. We'd like to know that that signup came from the google ad using the original UTM tags.

Currently, if you click on the ad, the marketing site show the source from the UTM tags, but the app site is not showing any source. If we click on a link that goes directly to the app site (skipping the marketing site), Plausible does show the UTM tags as the source.

Is this possible? Not sure we're doing something wrong or if it's just not possible.

**Comment by metmarkosaric** [comment]
Any visit from https://example.com to https://go.example.com will have https://example.com as the source (or whatever UTM tags you have in the link on https://example.com) rather than the initial source of how the visitor found https://example.com in the first place. When you're using a different Plausible snippet, it is all isolated from each other so no way to continue tracking the same session between different Plausible snippets.

If you want to see the original source, you could use the same Plausible snippet on both sites. There are some negatives of that though that you can read about here https://plausible.io/docs/plausible-script#can-i-use-the-same-snippet-across-my-domain-and-subdomain

You could also send some stats to both dashboards at the same time which will allow you to keep the sites on their own independent dashboards too. See https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time

**Reply by treebetty** [reply]
Thanks for the info Marko! Will see what works best.
